gamestonk_terminal/cryptocurrency/defi/defi_controller.py

```diff
@@ -165,6 +189,275 @@ def call_ayr(self, other_args: List[str]):
+            "-a",
+            "--address",
+            dest="address",
+            type=str,
+            help="Terra address. Valid terra addresses start with 'terra'",
+            required="-h" not in other_args,
+        )
+        parser.add_argument(
+            "-l",
+            "--limit",
+            dest="limit",
+            type=check_positive,
+            help="Number of delegations",
+            default=10,
+        )
+
+        if other_args and not other_args[0][0] == "-":
+            other_args.insert(0, "-a")
+
+        ns_parser = parse_known_args_and_warn(
+            parser, other_args, EXPORT_ONLY_RAW_DATA_ALLOWED
+        )
+
+        if ns_parser:
+            terramoney_fcd_view.display_account_staking_info(
+                export=ns_parser.export, address=ns_parser.address, top=ns_parser.limit
+            )
+
+    def call_validators(self, other_args: List[str]):
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="validators",
+            description="""
+                Displays information about terra validators.
+                [Source: https://fcd.terra.dev/swagger]
+            """,
+        )
+        parser.add_argument(
+            "-l",
+            "--limit",
+            dest="limit",
+            type=check_positive,
+            help="Number of validators to show",
+            default=10,
+        )
+        parser.add_argument(
+            "-s",
+            "--sort",
+            dest="sortby",
+            type=str,
+            help="Sort by given column. Default: votingPower",
+            default="votingPower",
+            choices=[
+                "validatorName",
+                "tokensAmount",
+                "votingPower",
+                "commissionRate",
+                "status",
+                "uptime",
+            ],
+        )
+
+        parser.add_argument(
+            "--descend",
+            action="store_false",
+            help="Flag to sort in descending order (lowest first)",
+            dest="descend",
+            default=False,
+        )
+
+        ns_parser = parse_known_args_and_warn(
+            parser, other_args, EXPORT_ONLY_RAW_DATA_ALLOWED
+        )
+
+        if ns_parser:
+            terramoney_fcd_view.display_validators(
+                export=ns_parser.export,
+                sortby=ns_parser.sortby,
+                descend=ns_parser.descend,
+                top=ns_parser.limit,
+            )
+
+    def call_govp(self, other_args: List[str]):
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="govp",
+            description="""
+                Displays terra blockchain governance proposals list.
+                [Source: https://fcd.terra.dev/swagger]
+            """,
+        )
+        parser.add_argument(
```

Comment:
this is only returning 4 proposals. just to be sure, does it mean this commands returns only the active proposals?

Reply:
yes, it returns only active proposals, but anyway I could add maybe one additional option: `all` and it will display proposals without filters applied. What do you think ?

Reply:
nice! agree!